includes/class.llms.emails.php

```diff
@@ -138,13 +138,12 @@ public function get_css( $rule = '', $echo = true ) {
 		if ( isset( $css[ $rule ] ) ) {
 
 			if ( $echo ) {
-				echo $css[ $rule ];
+				echo esc_attr( $css[ $rule ] );
```

Comment:
I don't know what CSS rules are being used here, but I feel esc_attr could break certain styles yeah?

If we ensure earlier that the values used to generated the rules are sanitized, we can ignore the escape check here.